gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/PolicyPluginService.java

```diff
@@ -40,8 +41,16 @@ public interface PolicyPluginService extends PluginService<PolicyPluginEntity> {
     /**
      * Get the schema form for the given policy plugin
      * @param policyPluginId is the id of the policy
-     * @param schemaDisplayFormat the format of the schema to return
```

Comment:
WDYT about keeping the `@param schemaDisplayFormat the format of the schema to return` description in the java doc for both methods in this service?